.github/workflows/ci_cron_weekly.yml

```diff
@@ -1,47 +1,100 @@
 name: Weekly cron
 
+# on:
+#   schedule:
+#     # run every Monday at 6am UTC
+#     - cron: '0 6 * * 1'
 on:
-  schedule:
-    # run every Monday at 6am UTC
-    - cron: '0 6 * * 1'
+  push:
+  pull_request:
 
 jobs:
-  tests:
-    name: ${{ matrix.name }}
-    runs-on: ${{ matrix.os }}
+  # tests:
+  #   name: ${{ matrix.name }}
+  #   runs-on: ${{ matrix.os }}
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       include:
+
+  #         - name: Documentation link check
+  #           os: ubuntu-latest
+  #           python: 3.7
+  #           toxenv: linkcheck
+
+  #         # TODO: Uncomment when 3.10 is more mature. Should we use devdeps?
+  #         # Test against Python dev in cron job.
+  #         #- name: Python dev with basic dependencies
+  #         #  os: ubuntu-latest
+  #         #  python: 3.10-dev
+  #         #  toxenv: pydev-test
+
+  #   steps:
+  #   - name: Checkout code
+  #     uses: actions/checkout@v2
+  #     with:
+  #       fetch-depth: 0
+  #   - name: Set up Python
+  #     uses: actions/setup-python@v2
+  #     with:
+  #       python-version: ${{ matrix.python }}
+  #   - name: Install language-pack-de and tzdata
+  #     if: ${{ matrix.os == 'ubuntu-latest' }}
+  #     run: sudo apt-get install language-pack-de tzdata
+  #   - name: Install graphviz
+  #     if: ${{ matrix.toxenv == 'linkcheck' }}
+  #     run: sudo apt-get install graphviz
+  #   - name: Install Python dependencies
+  #     run: python -m pip install --upgrade tox codecov
+  #   - name: Run tests
+  #     run: tox ${{ matrix.toxargs}} -e ${{ matrix.toxenv}} -- ${{ matrix.toxposargs}}
+
+
+  tests_more_architectures:
+
+    # The following architectures are emulated and are therefore slow, so
+    # we include them just in the weekly cron. These also serve as a test
+    # of using system libraries and using pytest directly.
+
+    runs-on: ubuntu-18.04
+    name: Python 3.7 on ${{ matrix.arch }}
+
     strategy:
       fail-fast: false
       matrix:
         include:
+          - arch: aarch64
+          - arch: ppc64le
+          - arch: s390x
 
-          - name: Documentation link check
-            os: ubuntu-latest
-            python: 3.7
-            toxenv: linkcheck
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Run tests
+        id: build
+        with:
+          arch: ${{ matrix.arch }}
+          distro: buster
 
-          # TODO: Uncomment when 3.10 is more mature. Should we use devdeps?
-          # Test against Python dev in cron job.
-          #- name: Python dev with basic dependencies
-          #  os: ubuntu-latest
-          #  python: 3.10-dev
-          #  toxenv: pydev-test
+          shell: /bin/sh
```

Comment:
@mhvk thanks for investigating! I probably need to change this then :) Will try shortly.

Reply:
Yes, that might just be it!

Reply:
`/bin/bash` should exist, since it is a Debian system